insalan/urls.py

```diff
@@ -32,9 +31,12 @@
     path("v1/tickets/", include("insalan.tickets.urls")),
     path("v1/langate/authenticate", langate_views.LangateUserView.as_view()),
     path("v1/content/", include("insalan.cms.urls")),
-    #path("v1/api-auth/", include("rest_framework.urls", namespace="rest_framework")),
-    path("v1/admin/", admin.site.urls),
+    path("v1/admin/", admin.site.urls)
+
 ]
+if not int(getenv("DEV")):
```

Comment:
You forgot to add a default for `getenv`, which is currently making the CI fail (remember that `int` expects a string)

Reply:
this should be fine now